scripts/hypothesistooling.py

```diff
@@ -57,6 +57,16 @@ def latest_version():
     return latest
 
 
+def hash_for_name(name):
+    return subprocess.check_output([
+        "git", "rev-parse", name
+    ]).decode('ascii').strip()
+
+
+def on_master():
+    return hash_for_name("HEAD") == hash_for_name("origin/master")
```

Comment:
I’ve been doing Travis stuff today, and got slightly bogged down because Travis does funky things to Git. Do we need to worry about that here? https://graysonkoonce.com/getting-the-current-branch-name-during-a-pull-request-in-travis-ci/